subsys/bluetooth/audio/shell/bap.c

```diff
@@ -3609,16 +3609,17 @@ static int cmd_context(const struct shell *sh, size_t argc, char *argv[])
 		return err;
 	}
 
-	if (ctx_val == BT_AUDIO_CONTEXT_TYPE_PROHIBITED ||
-	    ctx_val > BT_AUDIO_CONTEXT_TYPE_ANY) {
-		shell_error(sh, "Invalid context: %lu", ctx_val);
-
-		return -ENOEXEC;
-	}
 
 	ctx = ctx_val;
 
 	if (!strcmp(argv[3], "supported")) {
+		if (ctx_val == BT_AUDIO_CONTEXT_TYPE_PROHIBITED ||
+		ctx_val > BT_AUDIO_CONTEXT_TYPE_ANY) {
```

Comment:
The `ctx_val > BT_AUDIO_CONTEXT_TYPE_ANY` should still apply for both. Only the `ctx_val == BT_AUDIO_CONTEXT_TYPE_PROHIBITED` check should be for the supported contexts